docs/UserGuide.adoc

```diff
@@ -87,33 +87,60 @@ image::autocomplete.png[width="500"]
 
 === Viewing help : `help` or `h`
 
-If you are feeling lost, enter `help` or `h` in Medeina's command box, or press kbd:[Enter] to bring up this user guide document in a pop-up window.
+If you are feeling lost, enter `help` or `h` in Medeina's command box, or press kbd:[F1] to bring up this user guide document in a pop-up window.
 
 === Adding a contact, pet patient or appointment : `add` or `a` `[Since v1.2]`
 
-You can use the `add` or `a` to do the following :
+You can use `add` or `a` to do the following :
 
-To add a new contact:: `add **-o** n/CONTACT_NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS nr/NRIC [t/TAG]...`
+[width="100%", cols="15%,<45%a,<40%", options="header"]
+|=====
+| To add
+| Command syntax
+| Remarks
 
-To add a new pet patient under an existing contact (owner):: `add **-p** n/PET_PATIENT_NAME s/SPECIES b/BREED c/COLOR bt/BLOOD_TYPE [t/TAG]... **-o** nr/OWNER_NRIC`
+.^a|Person
+| `add **-o** n/CONTACT_NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS nr/NRIC [t/TAG]...`
+a| * Tag is optional.
 
-To add a new appointment for an existing pet patient:: `add **-a** d/YYYY-MM-DD HH:MM r/REMARK [t/TAG]... **-o** nr/OWNER_NRIC **-p** n/PET_PATIENT_NAME`
+.^| Pet patient
+| `add **-p** n/PET_PATIENT_NAME s/SPECIES b/BREED c/COLOR bt/BLOOD_TYPE [t/TAG]... **-o** nr/OWNER_NRIC`
+a| * OWNER_NRIC must belong to an existing contact.
+* Tag is optional.
 
-To add a new appointment for a new pet patient under a new owner:: `add **-o** n/CONTACT_NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS nr/NRIC [t/TAG]... **-p** n/PET_PATIENT_NAME s/SPECIES b/BREED c/COLOR bt/BLOOD_TYPE [t/TAG]... **-a** d/YYYY-MM-DD HH:MM r/REMARK [t/TAG]...`
+.^| Appointment
+| `add **-a** d/YYYY-MM-DD HH:MM r/REMARK t/TAG...` +
+**-o** nr/OWNER_NRIC **-p** n/PET_PATIENT_NAME
+a| * OWNER_NRIC must belong to an existing contact.
+* PET_PATIENT_NAME must belong to an existing pet patient.
+* Appointment time HH:MM must be in 30-minute blocks.
+* Compulsory tag (minimum 1).
 
-[TIP]
-Contact, pet patient and appointment can have any number of tags (including 0)
+.^| All of the above
+| `add **-o** n/CONTACT_NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS nr/NRIC [t/TAG]...` +
+`**-p** n/PET_PATIENT_NAME s/SPECIES b/BREED c/COLOR bt/BLOOD_TYPE [t/TAG]...` +
+`**-a** d/YYYY-MM-DD HH:MM r/REMARK [t/TAG]...`
+a|
+|=====
+
+{empty} +
 
 [NOTE]
 Do not leave REMARK empty. You can specify '-' or 'nil' if there is no remark for the appointment.
 
-Examples:
+{empty} +
+
+image::add_examples.PNG[width="700"]
```

Comment:
I like the new command better, as it shows the fields very clearly. However, I think the examples in text are also important, because users might want to just copy + paste into the application to test them out. Maybe we can do a new section called command samples? Then include all the commands within examples in text. Do you think that will be better?

Reply:
Like a table to show command, command format and sample command?

Reply:
Sounds good, I think in addition we can have a hyperlink to the bottom where there is a command for add, then hyperlink back to add (refer to add) etc.?